tests/test_swc.py

```diff
@@ -49,9 +50,9 @@
 paths_s3 = swc.graph_to_paths(G_s3)
 
 # create a subset of the dataframe
-url = "s3://mouse-light-viz/precomputed_volumes/brain1"
-mip = 6
-ngl = NeuroglancerSession(url, mip=mip, url_segments=url + "_segments")
+# url = "s3://mouse-light-viz/precomputed_volumes/brain1"
```

Comment:
Change to a variable name that is more descriptive and matches the earlier variable that you called "s3_path"